tests/test_data_mapping_utils.py

```diff
@@ -0,0 +1,37 @@
+from pytest_mock import mocker
+from bia_export.data_mapping_utils import create_export_image
+
+
+from .utils import (
+    get_template_api_biosample,
+    get_template_api_image_acquisition,
+    get_template_api_specimen,
+    get_template_api_study,
+    get_template_api_image,
+    get_template_export_image,
+    add_image_representation,
+    get_template_ome_zarr_image,
+)
+
+
+def test_create_export_image(mocker):
+    # We patch ome_zarr_image_from_ome_zarr_uri in bia_export.data_mapping_utils even though it's defined in
+    # bia_export.proxyimage. This is because we have to replace the method as it's being imported into bia_export.data_mapping_utils
+    # so that when create_export_image is called it only has access the replacement.
+    # Knowing where to patch is tricky, see: https://docs.python.org/3/library/unittest.mock.html#where-to-patch
+    mocker.patch(
```

Comment:
Maybe not now if urgent, but it might be worth thinking through how to get "test fixture data" with the api working. Mongo supports adding preexisting data volumes at startup so it should all be mostly wiring things together.

IMO mocks for things that are deeper in the code are non-obvious, it might be worth looking if an actual api instance is feasible (should be, maybe a little bit slower) and we get some more invariants (always matching models, automatically adding defaults) for free